example/user-guides/alerting/prometheus-example-rules.yaml

```diff
@@ -12,3 +12,19 @@ spec:
     rules:
     - alert: ExampleAlert
       expr: vector(1)
+  - name: node-status
+    rules:
+    - alert: DiskPressureWarning
+      expr: sum(kube_node_status_condition{condition='DiskPressure', status='true'} == 1) > 0
```

Comment:
It assumes that Prometheus collects the kube_node_* metrics which may not be the case. I suppose that your idea is to show an example with alerts at different severities?